pkg/certificates/bootstrap/file.go

```diff
@@ -0,0 +1,174 @@
+package bootstrap
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"kubevirt.io/client-go/log"
+
+	"kubevirt.io/kubevirt/pkg/certificates/triple/cert"
+)
+
+type FileCertificateManager struct {
+	stopCh         chan struct{}
+	certAccessLock sync.Mutex
+	stopped        bool
+	cert           *tls.Certificate
+	caCert         *x509.Certificate
+	caBundle       *x509.CertPool
+	certDir        string
+}
+
+func NewFileCertificateManager(certDir string) *FileCertificateManager {
+	return &FileCertificateManager{certDir: certDir}
+}
+
+func (f *FileCertificateManager) Start() {
+	objectUpdated := make(chan struct{}, 1)
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.DefaultLogger().Reason(err).Critical("Failed to create an inotify watcher")
```

Comment:
Is "Critical" the same as "Fatal". i just want to make sure this is treated like an assert